src/content/authors/evert.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: Evert
-avatar: ./assets/tjs.png
-avatarAlt: TorontoJS Icon
+name: Evert Pot
+avatar: https://s.gravatar.com/avatar/9e9b4f80c65dc6076d74cd421084e4fc?s=400
```

Comment:
This broke the build. I tried to update the `Avatar` component to support absolute URLs not using the assets system, but I'm a little lost on how to approach this. Happy to do the work but could use a pointer!

Reply:
So, the `Avatar` component actually only displays the image. The data comes from a `getEntry` call on the `AuthorCard` component.

That call will read the markdown file and run a schema validation on that data. (Astro also does schema validation for the whole project before building)

The problem is with the schema, we are using the [`image` method](https://docs.astro.build/en/guides/images/#images-in-content-collections) to set the schema for the `avatar` field. It expects a local file instead of a url.

There are a couple solutions to that:
1. Add a wrapper around the `getEntry` call and change the schema to accept a string, then [manually process the image loading](https://docs.astro.build/en/guides/images/#generating-images-with-getimage). (See `utils` folder for examples of wrappers).
2. Move to Astro's new way of loading images [Image Services](https://docs.astro.build/en/reference/image-service-reference/)

I believe you want to keep the image as an external url right, not bring it as a local image?
To do that, we would need to create some pre processing.

Reply:
> I believe you want to keep the image as an external url right, not bring it as a local image?
> To do that, we would need to create some pre processing. Take a look at the calls to > import.meta.glob and then to getImage under src/pages/feed.xml.ts

Yes, this would be my preference! I'll try to see if I can change this, but if I get stuck I'll just upload a jpeg =)

Reply:
Let me know if you need help with anything.

The main issue is we need to use the `import.meta.glob` to let astro know about an asset to load, or else we get broken images/assets.

Reply:
I've pushed a fix for this on the branch `feat/remote-avatar-url`. Try merging it and see if this will solve the problem.

Reply:
It did ! Thank you!!

Reply:
Awesome! Resolving this conversation 🥳